OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -299,18 +339,48 @@ open class OptimoveConfigBuilder: NSObject {
             }
             Logger.info("\(OptimobileConfig.self) building skipped.")
             return nil
-        }()
+        }()    
+
+        let preferenceCenterConfig: PreferenceCenterConfig? = features.contains(.preferenceCenter) ? getPreferenceCenterConfig() : nil
```

Comment:
also this line i think doesnt work (?). 

if we use delayed initialisation containing PC in feature set, we would try to get config here. But because it's a delayed init we dont have credentials -- it would log error. But it's not an error, it's a legid delayed init